infrastructure/src/main/java/org/corfudb/infrastructure/SequencerServer.java

```diff
@@ -332,17 +344,23 @@ public void trimCache(CorfuPayloadMsg<Long> msg, ChannelHandlerContext ctx, ISer
             cache.invalidateUpTo(trimMark);
         }
 
+        // Remove trimmed addresses from each address map and set new trim mark
+        for(StreamAddressSpace streamAddressSpace : streamsAddressSpaceMap.values()) {
+            streamAddressSpace.trim(trimMark);
```

Comment:
StreamAddressSpace doesn't seem to belong to the wire protocol ?

Reply:
Yes, you're right moved it under /runtime/stream